tasks/unit_tests/github_tasks_tests.py

```diff
@@ -112,3 +112,48 @@ def test_invalid_team_label(self):
         expected_labels = []
 
         self.make_test(changed_files, expected_labels, possible_labels=['team/team-doc'])
+
+
+class TestExtractQADescriptionFromPR(unittest.TestCase):
```

Comment:
You can maybe add:
- tests where the `test/QA` section is the first or last of the description to check "edge cases"
- a test with the `test/QA` section content having special characters (eg datadog-app urls, or `*,$,?`)

Reply:
Adding a full test

Reply:
Not sure I understand what you mean by 

> tests where the test/QA section is the first or last of the description to check "edge cases"

Do you mean editing the template? Editing the template should break this test, that I expect is executed on template changes. Of course folks can edit the content of the description, but I expect that not to happen often

Reply:
Yes I was thinking about folks editing their description and removing part of the template (so for instance have only a partial version of it).